lib/cdo/aws/s3.rb

```diff
@@ -59,15 +59,17 @@ def self.connect_v2!
                     Aws::S3::Client.new
                   end
 
-      # Adjust s3_timeout using a dynamic variable,
-      # updating the S3 client if the variable changes.
+      # Adjust the timeouts for opening an HTTP connection and waiting for a response to short values to ensure we fail
+      # fast when S3 is slow instead tying up the fixed number of HTTP worker threads we provision per vCPU.
       timeout = DCDO.get('s3_timeout', 15)
-      if timeout != s3.config.http_open_timeout
+      connection_pool_timeout = DCDO.get('s3_connection_pool_timeout', 5)
```

Comment:
The default is technically a change from where we are at now (I think it's `1`)? Should we keep it consistent so when it rolls out we don't see something that may arise from the default being used?